### ISSUE TITLE
colorAmbient in Blender generated JSON files isn't being handled

### ISSUE THREAD

**Issue by Programmer-D** [mrdoob/three.js#7486]
Versions:
**Three.js:** Updated from r71 to r73
**Three.js blender exporter:** Updated from 1.4.0 to 1.5.0
**Blender:** 2.76

Since I updated the Blender exporter and three.js, i decided to reexport my models. I now get the following **error** after loading the models:

```javascript
Loader.createMaterial: Unsupported colorAmbient [0.64, 0.64, 0.64]
THREE.Loader.createMaterial @ three.js:14584
THREE.Loader.initMaterials @ three.js:14377
THREE.JSONLoader.parse @ JSONLoader.js:538
(anonymous function) @ JSONLoader.js:72
(anonymous function) @ three.js:14683
```

colorAmbient was present in my models before, and I never got this error in r71. I did a quick search in the r73 three.js code, but I didn't find a reference to it anywhere. The case statement where this error is generated from doesn't check for colorAmbient. As a test, I removed colorAmbient from the JSON files, and I didn't get the above error anymore.

Should colorAmbient be handled in three.js, or should it not be generated and put into the JSON files at all from the blender exporter?

**Comment by tschw** [comment]
Double-check whether the exporter is really from version r73.

**Comment by Programmer-D** [comment]
It is. I actually completely reinstalled Blender to make sure there was no hint of the old exporter, and I now only have the r73 build, so that's the only exporter I could have put back into my new Blender installation. I also removed everything in the **AppData\Roaming\Blender Foundation** folder since that stuff didn't get removed when I installed Blender.

Were you not able to reproduce the problem?

**Comment by tschw** [comment]
Nope. Mine is on the `dev` branch, though.

**Comment by mrdoob** [comment]
> Should colorAmbient be handled in three.js, or should it not be generated and put into the JSON files at all from the blender exporter?

It shouldn't be in the json. I think this has been fixed in `dev` already.

**Comment by tschw** [comment]
Yes, it's #7374

**Comment by Programmer-D** [comment]
I checked out the dev branch and have confirmed that it is fixed there. r73 was tagged on Thursday, October 15, 2015 9:30:46 PM, and it appears the the fix was made on 10/18/2015 8:59:30 PM in revision d31d47352a0febb4540a26e916c26324a0189382, which shows the fix came after the release of r73.

I think this issue can be closed, but users should just be aware that they should pull the dev branch like I did if they want to use the blender exporter before r74 is released.
